uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/absence-tab.js

```diff
@@ -0,0 +1,27 @@
+(function (CRM, require) {
+  CRM.vars.leaveAndAbsences = {
+    baseURL: CRM.absenceTabApp.path
+  };
+
+  var srcPath = CRM.vars.leaveAndAbsences.baseURL + '/js/angular/src/leave-absences';
```

Comment:
> Not sure what you mean by backend

You're doing this in your PR
```php
return array(
  'absenceTabApp' => array(
    'contactId' => CRM_Utils_Request::retrieve('cid', 'Integer'),
    'path' => CRM_Core_Resources::singleton()->getUrl('uk.co.compucorp.civicrm.hrleaveandabsences'),
  )
);
```

so why do you define `absenceTabApp` instead of directly `leaveAndAbsences`, since it's the property that you're using anyway?